### PR TITLE
Updated JSON schema for tide content collection.

### PR DESCRIPTION
**Pull request by joshua-salsadigital**
Included configurations for 
```
skipToResultLink
SkipToResultLinkLabel
model
elasticsearch-aggregation-order
elasticsearch-aggregation-size
elasticsearch-aggregation-show-count
```

On-behalf-of: @salsadigitalauorg (joshua.fernandes@salsadigital.com.au)

### CODE REVIEW

File: modules/tide_content_collection/json/schema.json [hunk 1]
```diff
@@ -181,6 +181,14 @@
...
           "description": "Store the state of the user interactions in the URL. Only one Content Collection can use this per page."
         },
+        "skipToResultLink": {
+          "type": "boolean",
+          "description": "Show skip to result link when accessing using a tab key, default is true."
+        },
+        "SkipToResultLinkLabel": {
+          "type": "string",
+          "description": "Default is `Skip to results`"
+        },
```
Comment (kepol) [inline modules/tide_content_collection/json/schema.json:191]: Is there some reason that `skipToResultLink` starts with a lowercase letter and `SkipToResultLinkLabel` starts with an uppercase letter? If not, these should be consistent.

File: modules/tide_content_collection/json/schema.json [hunk 2]
```diff
@@ -257,7 +265,14 @@
                     "type": "string"
                   },
                   "options": {
-                    "type": "object"
+                    "type": "object",
+                    "properties": {
+                      "model": {
+                        "type": "string",
+                        "description": "The URL query key for this field, and a fallback if elasticsearch-field is undefined."
```
Comment (kepol) [inline modules/tide_content_collection/json/schema.json:272]: Nitpick: Is `elasticsearch-field` used as a "word". Perhaps better just as "the Elasticsearch field".

**Review by kepol — COMMENTED** [review]
Thanks. Some general feedback.

File: modules/tide_content_collection/json/schema.json (hunk 2 above)
Comment (joshua-salsadigital) [reply modules/tide_content_collection/json/schema.json:272]: The above field references to 
elasticsearch-field 
https://github.com/salsadigitalauorg/content-collection-component/blob/master/src/validation/schema.json#L283

File: modules/tide_content_collection/json/schema.json (hunk 1 above)
Comment (joshua-salsadigital) [reply modules/tide_content_collection/json/schema.json:191]: https://github.com/dpc-sdp/ripple/blob/develop/packages/ripple-nuxt-tide/modules/landing-page/lib/content-collection.js#L23

**Review by sonnykt — APPROVED** [review]

**Comment by sonnykt** [comment]
Hi @MdNadimHossain this change is for https://github.com/dpc-sdp/tide_api/pull/69 (Raw JSON widget) which is in use by both content-vic and content-health. Please merge it when you are ready for the release.

**Comment by MdNadimHossain** [comment]
Hi @sonnykt 
is there any related ticket just to track the changes, so that I can just pass it to Jess saying this is the work that is getting merged as well !

**Comment by sonnykt** [comment]
> Hi @sonnykt
> is there any related ticket just to track the changes, so that I can just pass it to Jess saying this is the work that is getting merged as well !

I don't think there is a ticket for this one. This PR is to update the JSON schema bundled with the RAW JSON widget so that it could validate the latest JSON string provided by FE.

**Review by MdNadimHossain — APPROVED** [review]

**Pull request merged**